x/oracle/abci.go

```diff
@@ -40,6 +40,14 @@ func EndBlocker(ctx sdk.Context, k keeper.Keeper) error {
 		}
 
 		k.ClearExchangeRates(ctx)
+		// Clear median and median deviations
+		if isPeriodLastBlock(ctx, params.MedianPeriod) && experimental {
+			for _, v := range params.HistoricAcceptList {
```

Comment:
If an asset is removed from the Accept list will its last median be removed for exists forever? Maybe we should use the `KVStorePrefixIterator` like ClearExchangeRates does so that all medians are removed no matter what?

Reply:
Agreed!